build/templates/_converters.py.mako

```diff
@@ -54,6 +54,59 @@ def convert_timedelta_to_microseconds(value, library_type):
     return _convert_timedelta(value, library_type, 1000000)
 
 
+# This converter is not called from the normal codegen path for function. Instead it is
```

Comment:
Why not put this as part of the docstring?

Reply:
Because I didn't think the information in the comment was appropriate for a docstring.

I can move into one if you think it is.